src/wizard/wizard.scss

```diff
@@ -2,168 +2,147 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards {
+@mixin iui-wizard {
   @include iui-reset;
   @include iui-font-family;
 
-  > .iui-wizards-wrapper {
+  > .iui-wizard-container {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
     align-items: flex-start;
     width: 100%;
     margin-bottom: $iui-baseline * 0.5;
 
-    > .iui-wizards-step {
+    .iui-wizard-step {
+      flex: 1 1 0;
       align-items: center;
       display: flex;
-      flex-direction: column-reverse;
+      flex-direction: column;
       font-size: $iui-font-size-small;
       font-weight: $iui-font-weight-normal;
       line-height: $iui-line-height;
       position: relative;
-      width: 20%;
-
       @include themed {
         color: t(iui-color-background-positive);
       }
 
```

Comment:
Should we try to reduce nesting? E.g. `.iui-step-track` is 4 levels deep.

Reply:
I tried reducing the nesting in my latest push.  Let me know if you think I should keep reducing or if this is good enough.  cc @mayank99

Reply:
When in doubt, just look at the compiled css.
![image](https://user-images.githubusercontent.com/9084735/128237258-5f5d659e-63de-4692-b852-6db8cb9650e2.png)

Seems you should be able to remove the top level nesting (i.e. `.iui-wizard`) from all selectors.
After that, I don't think the `.iui-step-name` needs to be inside `.iui-wizard-step`. And `.iui-step-track` does not need to be inside `.iui-wizard-step`.

And so on...

Reply:
Should we add `iui-wizard` prefix to each class here?

Reply:
I've made some class name changes and tried lowering specificity.

Reply:
changing classnames without reducing nesting does not lower specificity
![image](https://user-images.githubusercontent.com/9084735/128411922-ee39e188-532a-4ebd-9b3d-cf70c92a7a10.png)

Reply:
I did reduce nesting as well, that's why I renamed the classes so by themselves they make a little more sense.  There are some parts that are still nested, I'm unsure if they can be un-nested & still work correctly.

Reply:
I think you can remove the top level `iui-wizard` from all places, no?

Reply:
Much better now!
![image](https://user-images.githubusercontent.com/9084735/128909106-416b9633-5aee-4ee6-80ff-1f54871bf24d.png)